storage/innobase/handler/ha_innodb.cc

```diff
@@ -1346,6 +1346,24 @@ innodb_enable_monitor_at_startup(
 /*=============================*/
 	char*	str);	/*!< in: monitor counter enable list */
 
+/** Checks if buffer pool is big enough to enable backoff algorithm.
```

Comment:
"Check if enabling given empty free list algorithm is allowed"

The header comment slightly differs between this declaration and the actual definition. I'd just move definition higher in the file so that there is no need for the declaration